### PR TITLE
Zendesk integration

### PR DESCRIPTION
**Pull request by stevegraham**
Needs to be finished and tests

### CODE REVIEW

File: app/controllers/feedback_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class FeedbackController < ApplicationController
+  def create
+    resource.assign_attributes params[:feedback]
+    resource.save
+
+    redirect_to request.referer, notice: "lel"
```
Comment (stevegraham) [inline app/controllers/feedback_controller.rb]: lel #FIXME

**Comment by stevegraham** [comment]
Thought I'd push this to a remote branch before I disappear.

File: app/forms/form/core.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
+class Form
```
Comment (stevegraham) [inline app/forms/form/core.rb]: Codeclimate no like

**Comment by stevegraham** [comment]
Needs front end styling.

This commit removes `.env` from the repo. You'll have to add the Zendesk creds to `.env` yourself.

@Demwunz needs front end. Please also style the flash messages too.

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -41,6 +41,7 @@
   main#content(role="main")
     .container
         p.flash.alert = flash[:alert]
+        p.flash.notice = flash[:notice]
```
Comment (stevegraham) [inline app/views/layouts/application.html.slim]: @Demwunz are these styled? Please style if not.

**Comment by stevegraham** [comment]
@dan-palmer please review code.

File: spec/helpers/feedback_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# describe FeedbackHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe FeedbackHelper, :type => :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (stevegraham) [inline spec/helpers/feedback_helper_spec.rb]: such delete plz

File: app/forms/form/core.rb (hunk 2 above)
Comment (dan-palmer) [inline app/forms/form/core.rb]: Why it no like? Form looks much tidier now.

File: app/helpers/feedback_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module FeedbackHelper
```
Comment (dan-palmer) [inline app/helpers/feedback_helper.rb]: We can bin this module

File: spec/views/feedback/new.html.slim_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "feedback/new.html.slim", :type => :view do
```
Comment (dan-palmer) [inline spec/views/feedback/new.html.slim_spec.rb]: :door: :wave:

**Comment by dan-palmer** [comment]
I'm happy with this :+1: nice job :wine_glass: 

@Demwunz did you put the magic touch on it?

**Comment by helenmott** [comment]
Test API key
LS82Sntzl0LXUDtTvykFUG1SEQLaKdupLoLtcAo5
group ID 22546296

**Comment by stevegraham** [comment]
such ready to roll after such designs

**Comment by stevegraham** [comment]
front end kiss and cuddle

**Pull request merged**